packages/components/tooltip/src/trigger.vue

```diff
@@ -89,7 +88,8 @@ export default defineComponent({
       stopWhenControlledOrDisabled,
       (e: KeyboardEvent) => {
         const { code } = e
-        if (code === EVENT_CODE.enter || code === EVENT_CODE.space) {
+        if (props.triggerKeys.includes(code)) {
+          e.preventDefault()
```

Comment:
@opengraphica Excuse me, why use `preventDefault` here, is there any special consideration?

This seems to cause the component contained in the Tooltip to be unable to use the Enter  keys 
and Space keys. [Demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IEVsTWVzc2FnZSB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxuY29uc3QgdmFsdWUgPSByZWYoJycpXG5jb25zdCBoYW5kbGVDbGljayA9ICgpID0+IHtcbiAgRWxNZXNzYWdlKHtcbiAgICBtZXNzYWdlOiAnQ29uZ3JhdHMsIHRoaXMgaXMgYSBzdWNjZXNzIG1lc3NhZ2UuJyxcbiAgICB0eXBlOiAnc3VjY2VzcycsXG4gIH0pXG59XG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtdG9vbHRpcCBjb250ZW50PVwiY29udGVudFwiPlxuICAgIDxlbC1idXR0b24gQGNsaWNrPVwiaGFuZGxlQ2xpY2tcIj5CdXR0b248L2VsLWJ1dHRvbj5cbiAgPC9lbC10b29sdGlwPlxuICA8ZWwtdG9vbHRpcCBjb250ZW50PVwiY29udGVudFwiPlxuICAgIDxlbC1pbnB1dCB2LW1vZGVsPVwidmFsdWVcIiB0eXBlPVwidGV4dGFyZWFcIiAvPlxuICA8L2VsLXRvb2x0aXA+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)

Reply:
That code is working as intended, basically "which keys trigger the tooltip to show?"

You shouldn't have a button that runs multiple actions when you press enter, either it controls a tooltip, or it does some other action. That is why the preventDefault is used. If "Enter" should do something other than show the tooltip, `triggerKeys` should be set to an empty array by the developer.

That being said, the default configuration for a tooltip doesn't make much sense. For the `tooltip` role, the tooltip should show and be read out on focus of the button, not on click.

E.g. default behavior from what is typically expected from a tooltip would look like:

```<el-tooltip content="content" trigger="focus" :trigger-keys="[]">```

Problem is changing this default behavior would be a breaking change, that can't be done immediately.